Examples/outputSinDAC.py

```diff
@@ -1,3 +1,8 @@
+from __future__ import division
+from __future__ import print_function
+from builtins import range
+from builtins import object
+from past.utils import old_div
```

Comment:
This adds a new requirement that the [past](https://pypi.python.org/pypi/past/) package must be installed.